tests/conftest.py

```diff
@@ -426,3 +427,35 @@ def strptime(cls, date_string, fmt):
             return dates[f"{date_string}"]
 
     return Mockdatetime
+
+
+PATCH_DEFAULT = r"""
+# HG changeset patch
+# User Test User <test@example.com>
+# Date 0 0
+#      Thu Jan 01 00:00:00 1970 +0000
+# Diff Start Line 7
+add another file.
+diff --git a/test.txt b/test.txt
+--- a/test.txt
++++ b/test.txt
+@@ -1,1 +1,2 @@
+ TEST
++adding another line
+""".strip()
+
+
+@pytest.fixture
+def create_patch_revision(db):
+    """A fixture that fake uploads a patch"""
```

Comment:
While we're here, this docstring should be updated (likely still refers to S3 uploads from before the migration).